packages/web-components/src/select/select.styles.ts

```diff
@@ -0,0 +1,233 @@
+        color: ${accentForegroundCutRestBehavior.var};
+    }
+
+    :host([disabled]) {
+        cursor: ${disabledCursor};
+        opacity: var(--disabled-opacity);
+    }
+
+    :host([disabled]) .control {
+        cursor: ${disabledCursor};
+    }
+
+    :host([disabled]:hover) {
+        background: ${neutralFillStealthRestBehavior.var};
+        color: ${neutralForegroundRestBehavior.var};
+        fill: currentcolor;
+    }
+
+    :host(:not([disabled])) .control:active {
+        background: ${neutralFillInputActiveBehavior.var};
+        border-color: ${neutralOutlineActiveBehavior.var};
+    }
+
+    :host([open][position="above"]) .listbox,
+    :host([open][position="below"]) .control {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+    }
+
+    :host([open][position="above"]) .control,
+    :host([open][position="below"]) .listbox {
+        border-top-left-radius: 0;
+        border-top-right-radius: 0;
+    }
+
+    :host([open][position="above"]) .listbox {
+        border-bottom: 0;
+        bottom: calc(${heightNumber} * 1px);
+    }
+
+    :host([open][position="below"]) .listbox {
+        border-top: 0;
+        top: calc(${heightNumber} * 1px);
+    }
+
+    .selected-value {
+        font-family: var(--body-font);
+        flex: 1 1 auto;
+        text-align: start;
+    }
+
+    .indicator {
+        flex: 0 0 auto;
+        margin-inline-start: 1em;
+    }
+
+    slot[name="listbox"] {
+        display: none;
+        width: 100%;
+    }
+
+    :host([open]) slot[name="listbox"] {
+        display: flex;
+        position: absolute;
+        ${elevation}
+    }
+
+    .end {
+        margin-inline-start: auto;
+    }
+
+    .start,
+    .end,
+    .indicator,
+    ::slotted(svg) {
+        ${`` /* Glyph size is temporary - replace when glyph-size var is added */}
+        fill: currentcolor;
+        height: 1em;
+        min-height: calc(var(--design-unit) * 4px);
+        min-width: calc(var(--design-unit) * 4px);
+        width: 1em;
+    }
+
+    ::slotted([role="option"]) {
+        flex: 0 0 auto;
+    }
+
+`.withBehaviors(
+  accentFillHoverBehavior,
+  accentForegroundCutRestBehavior,
+  neutralOutlineActiveBehavior,
+  neutralOutlineHoverBehavior,
+  neutralOutlineRestBehavior,
+  forcedColorsStylesheetBehavior(
+    css`
+            :host([disabled]) {
+                border-color: ${SystemColors.GrayText};
+                background-color: ${SystemColors.ButtonFace};
+                color: ${SystemColors.GrayText};
+                opacity: 1;
+                forced-color-adjust: none;
+            }
+
+            :host([disabled]:hover) {
+                background: ${SystemColors.ButtonFace};
+            }
+
+            :host([disabled]) .control {
+                color: ${SystemColors.GrayText};
+                border-color: ${SystemColors.GrayText};
+            }
+
+            :host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]) {
```

Comment:
this selector should be,
`:host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]:not([disabled]))`